lib/block-supports/block-visibility.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Block visibility block support flag.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Render nothing if the block is hidden.
+ *
+ * @param  string $block_content Rendered block content.
+ * @param  array  $block         Block object.
+ * @return string                Filtered block content.
```

Comment:
```suggestion
 * @param string $block_content Rendered block content.
 * @param array  $block         Block object.
 * @return string Filtered block content.
```

Per WPCS https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods